.eslintrc

```diff
@@ -253,6 +256,91 @@
     "spaced-comment": [2, "always"], // http://eslint.org/docs/rules/spaced-comment
-    "arrow-spacing": [2, { "before": true, "after": true }] // https://eslint.org/docs/rules/arrow-spacing
+    "arrow-spacing": [2, { "before": true, "after": true }], // https://eslint.org/docs/rules/arrow-spacing
+
+    /**
+    * Additional rules to enable one by one
+    */
+    // "array-bracket-spacing": ["error", "never"],
+    // "array-callback-return": ["error", { "allowImplicit": true }],
+    // "arrow-body-style": ["error", "as-needed", { "requireReturnForObjectLiteral": false }],
+    // "arrow-parens": [ "error", "as-needed", { "requireForBlockBody": true }],
+    // "block-spacing": ["error", "always"],
+    // "computed-property-spacing": ["error", "never"],
+    // "dot-location": ["error", "property"],
+    // "function-paren-newline": ["error", "multiline"],
+    // "import/export": "error",
+    // "import/first": ["error", "absolute-first"],
+    // "import/newline-after-import": "error",
+    // "import/no-duplicates": "error",
+    // "import/no-mutable-exports": "error",
+    // "import/no-named-default": "error",
+    // "new-parens": "error",
+    // "newline-per-chained-call": ["error", { "ignoreChainWithDepth": 4 }],
+    // "no-await-in-loop": "error",
+    // "no-bitwise": "error",
+    // "no-case-declarations": "error",
+    // "no-confusing-arrow": ["error", { "allowParens": true }],
+    // "no-empty-pattern": "error",
+    // "no-lonely-if": "error",
+    // // This rule copied from Airbnb's config
+    // "no-mixed-operators": ["error", {
+    //   "groups": [
+    //     ["%", "**"],
+    //     ["%", "+"],
+    //     ["%", "-"],
+    //     ["%", "*"],
+    //     ["%", "/"],
+    //     ["**", "+"],
+    //     ["**", "-"],
+    //     ["**", "*"],
+    //     ["**", "/"],
+    //     ["&", "|", "^", "~", "<<", ">>", ">>>"],
+    //     ["==", "!=", "===", "!==", ">", ">=", "<", "<="],
+    //     ["&&", "||"],
+    //     ["in", "instanceof"]
+    //   ],
+    //   "allowSamePrecedence": false
+    // }],
+    // "no-multi-assign": ["error"],
+    // "no-multi-spaces": ["error", { "ignoreEOLComments": false }],
+    // "no-plusplus": "error",
+    // "no-prototype-builtins": "error",
+    // "no-tabs": "error",
+    // "no-undef-init": "error",
+    // "no-unneeded-ternary": ["error", { "defaultAssignment": false }],
+    // "no-unsafe-finally": "error",
+    // "no-useless-computed-key": "error",
+    // "no-useless-concat": "error",
+    // "no-useless-constructor": "error",
+    // "no-useless-escape": "error",
+    // "no-void": "error",
+    // "object-curly-newline": ["error", { "ObjectExpression": { "multiline": true, "consistent": true }, "ObjectPattern": { "multiline": true, "consistent": true } }],
+    // "object-property-newline": ["error", { "allowAllPropertiesOnSameLine": true }],
+    // "object-shorthand": ["error", "always", { "ignoreConstructors": false, "avoidQuotes": true }],
```

Comment:
This will be a big cleanup job but will make a lot of code more consistent. What is the thought behind using `avoidQuotes: true`?

Reply:
I copied this one from Airbnb without looking closely, but now that you bring it up, I think I'd actually prefer `avoidQuotes: false`. I typically use shorthand even if a property has to have quotation marks for special characters in it.